pynsee/sirene/get_sirene_data.py

```diff
@@ -11,11 +11,14 @@
 from pynsee.utils._make_dataframe_from_dict import _make_dataframe_from_dict
 from pynsee.sirene.SireneDataFrame import SireneDataFrame
 
+import logging
+logger = logging.getLogger(__name__)
 
 @lru_cache(maxsize=None)
 def _warning_get_data():
-    print(
-        "\n!!! This function may return personal data, please check and\n comply with the legal framework relating to personal data protection !!!"
+    logger.info(
```

Comment:
must be warning

Reply:
Idem

Reply:
I opened an issue #147 but in the meantime it's probably best not to change the current behavior where this is visible by default, though I agree that it is not how we should do it in the future